script.js

```diff
@@ -1,16 +1,18 @@
-addButton.onclick = function () {
+document.getElementById("addButton").onclick = function () {
```

Comment:
It's preferred to use single quotes in string literals. Double quotes should be used in HTML attributes like `<input id="inputTask" />`.